### PR TITLE
Add change player entity API

### PR DESCRIPTION
**Pull request by Creeperface01**

### CODE REVIEW

File: src/main/java/cn/nukkit/Player.java [hunk 1]
```diff
@@ -261,10 +260,42 @@ public boolean hasAutoJump() {
...
     @Override
     public void spawnTo(Player player) {
         if (this.spawned && player.spawned && this.isAlive() && player.isAlive() && player.getLevel().equals(this.level) && player.canSee(this) && !this.isSpectator()) {
-            super.spawnTo(player);
+            if(this.entityType == null) {
+                super.spawnTo(player);
+            } else if (!this.equals(player) && !this.hasSpawned.containsKey(player.getLoaderId())) {
+                this.hasSpawned.put(player.getLoaderId(), player);
+
+                if (this.skin.getData().length < 64 * 32 * 4) {
```
Comment (Pub4Game) [inline src/main/java/cn/nukkit/Player.java]: I may be wrong, but maybe there has to be <=

**Comment by Creeperface01** [comment]
@Pub4Game where?

**Comment by Pub4Game** [comment]
@Creeperface01 Anywhere :) I made a mistake

**Comment by Creeperface01** [comment]
ok :D nvm

**Comment by Pub4Game** [comment]
@Creeperface01 "nvm" what?

**Comment by Creeperface01** [comment]
NV = never
M = mind

xD

**Comment by Pub4Game** [comment]
233))

:smile:

**Comment by Creeperface01** [comment]
so? :D

**Comment by Creeperface01** [comment]
i think it is good because if you want e.g. despawn player and spawn another entity so it is too complicated :D

**Pull request closed without merging**